File: fizzbuzz.py
#!/usr/bin/python3

def main(n):
    for i in range(1, n+1):
        if i % 15 == 0:
            print("Fizz Buzz")
        elif i % 3 == 0:
            print("Fizz")
        elif i % 5 == 0:
            print("Buzz")
        else:
            print(i)


if __name__ == "__main__":
    import sys
    main(int(sys.argv[1]))

File: Octave/fizzbuzz.m
#!/usr/bin/octave -qf

n = str2num(argv(){1});

for i = 1 : n
    if mod(i, 15) == 0
        fprintf("Fizz Buzz\n")
    elseif mod(i, 3) == 0
        fprintf("Fizz\n")
    elseif mod(i, 5) == 0
        fprintf("Buzz\n")
    else
        fprintf('%d\n',i)
    end
end
